docs/reference/commandline/search.md

```diff
@@ -97,10 +97,10 @@ be in the range between 1 and 100. The default value of `--limit` is 25.
 
 ### Filtering
 
-The filtering flag (`-f` or `--filter`) format is a `key=value` pair. If there is more
-than one filter, then pass multiple flags (e.g. `--filter "foo=bar" --filter "bif=baz"`)
+The filtering flag (`-f` or `--filter`) format is a `key=value` pair. The supported key and value given below. If there is more
```

Comment:
I think the extra sentence here is a bit redundant, because the "The currently supported filters are" sentence below already describes that. I'd remove that change;

```suggestion
The filtering flag (`-f` or `--filter`) format is a `key=value` pair. If there is more
```